### ISSUE TITLE
Add repository instructions on how to build with gradle

### ISSUE THREAD

**Issue by HSQuillin** [BlueStaggo/VoxelThing#1]
In the `README.md`, there should be a step-by-step on how to build this project

**Comment by BlueStaggo** [comment]
Yeah, sounds like a good idea. Building is just as simple as `gradle :client:shadowJar` and running it should be as simple as clicking on the shadow jar for your platform of choice.

**Comment by BlueStaggo** [comment]
Done!
